x-pack/plugin/ql/test-fixtures/src/main/java/org/elasticsearch/xpack/ql/CsvSpecReader.java

```diff
@@ -44,7 +45,9 @@ public Object parse(String line) {
                     assertThat("Early schema already declared " + earlySchema, earlySchema.length(), is(0));
                     earlySchema.append(line.substring(SCHEMA_PREFIX.length()).trim());
                 } else if (line.toLowerCase(Locale.ROOT).startsWith("required_capability:")) {
-                    requiredCapabilities.add(line.substring("required_capability:".length()).trim());
+                    Arrays.stream(line.substring("required_capability:".length()).split(","))
```

Comment:
FYI: I'm expanding here to accept a comma separated list of capabilities. We could add just the "ignored" capability to the tests, and add it as another exception on CsvTests, as it can't run Metadata tests, but that would be _weird_ for the long term.

I don't like "required_capability:" being in singular, but I'm not sure migrating everything or adding another keyword would be better (?)